### PR TITLE
[Improvement] Add port checks for MySQL and Postgres

### PR DESCRIPTION
**Pull request by justinmclean**
Add scripts to check if ports are in use and that means the user is likely running MySQl or Posgres locally. I assume for this scripts to run new MySQL and Postgres docker images will need to be made. I tested the scripts manually and they detected if the port was in use.

### CODE REVIEW

**Comment by qqqttt123** [comment]
Think twice. The shell shouldn't take effect. The shell of `init.sh` should be executed in the Docker image container. The ports are available all the time.

File: launch-playground.sh [hunk 1]
```diff
@@ -15,6 +15,18 @@ else
   exit
 fi
 
+echo "Checking if MySQL is runing"
+if lsof -Pi :3306 -sTCP:LISTEN -t >/dev/null ; then
+    echo "MySQL may allready be running on port 3306. Please shut down MySQL before launching the playground."
+    exit
```
Comment (qqqttt123) [inline launch-playground.sh:21]: Do we need to exit with code -1, too?

**Comment by qqqttt123** [comment]
Is the scrip ok for both mac and linux?

File: launch-playground.sh (hunk 1 above)
Comment (justinmclean) [reply launch-playground.sh:21]: I don't think so other existing exit in the script doesn't exit with -1

**Comment by justinmclean** [comment]
> Is the script ok for both Mac and Linux?

Yep see https://manpages.ubuntu.com/manpages/bionic/en/man8/lsof.8.html

File: launch-playground.sh [hunk 2]
```diff
@@ -15,6 +15,18 @@ else
...
+fi
+
+echo "Checking if Postgres is runing"
+if lsof -Pi :5432 -sTCP:LISTEN -t >/dev/null ; then
+    echo "Postgres may allready be running on port 5432. Please shut down Postgres before launching the playground."
+    exit -1
```
Comment (qqqttt123) [inline launch-playground.sh:27]: Why do we exit with code `-1` here?

**Comment by shaofengshi** [comment]
This is something similar as https://github.com/apache/gravitino-playground/issues/60, I propose to solve that by chaning to an less possible conflict port in local, which is common used in docker deployments.

**Comment by shaofengshi** [comment]
Hi Justin, I have changed the playground to use non-default ports to avoid such conflicts, see: https://github.com/apache/gravitino-playground/commit/fd68dd2c59617451cf065f8e2e1d5397928464b3

So I think this PR is not needed; If user still got conflict (the possibility is very low), he can modify the yaml.

**Pull request closed without merging**